### PR TITLE
pythonPackages.dufte: 0.2.9 -> 0.2.12

### PR DESCRIPTION
**Pull request by risicle**
###### Motivation for this change
Bump.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/dufte/default.nix [hunk 1]
```diff
@@ -34,6 +34,10 @@ buildPythonPackage rec {
   '';
   checkInputs = [ pytestCheckHook ];
   pythonImportsCheck = [ "dufte" ];
+  pytestFlagsArray = [
+    # we don't have the "exdown" package (yet)
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/dufte/default.nix:38]: I am going to do a PR which adds exdown.

Comment (risicle) [reply pkgs/development/python-modules/dufte/default.nix:38]: Even so, I think this test just renders the readme and isn't useful enough to warrant the extra build-time dependency.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 111354` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
6 packages built:
</summary>
<ul>
<li>python37Packages.dufte</li>
<li>python37Packages.perfplot</li>
<li>python38Packages.dufte</li>
<li>python38Packages.perfplot</li>
<li>python39Packages.dufte</li>
<li>python39Packages.perfplot</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 111354` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
5 packages built:
</summary>
<ul>
<li>python37Packages.dufte</li>
<li>python37Packages.perfplot</li>
<li>python38Packages.dufte</li>
<li>python38Packages.perfplot</li>
<li>python39Packages.dufte</li>
</ul>
</details>

File: pkgs/development/python-modules/dufte/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/development/python-modules/dufte/default.nix:38]: Already did it so whatever.

**Pull request merged**